package/network/utils/comgt/files/3g.sh

```diff
@@ -63,11 +63,12 @@ proto_3g_setup() {
 				SIERRA=1
 			elif echo "$cardinfo" | grep -qi huawei; then
 				case "$service" in
-					umts_only) CODE="14,2";;
-					gprs_only) CODE="13,1";;
-					*) CODE="2,2";;
+					lte_only)  CODE="SYSCFGEX=03,3FFFFFFF,2,4,40,,";;
```

Comment:
Wrong. 40 means only LTE band 7. Should be 7FFFFFFFFFFFFFFF (all bands - more generic). See "AT Command Interface Specification" for Huawei devices.